src/main/java/com/minecolonies/coremod/commands/ActionArgument.java

```diff
@@ -69,4 +69,71 @@ public String getUsage()
         }
         return sb.toString();
     }
+
+    @Override
+    public int hashCode()
+    {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((actionArgumentList == null) ? 0 : actionArgumentList.hashCode());
+        result = prime * result + ((name == null) ? 0 : name.hashCode());
+        result = prime * result + (required ? 1231 : 1237);
+        result = prime * result + ((type == null) ? 0 : type.hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj)
+    {
+        if (this == obj)
+        {
+            return true;
+        }
+        if (obj == null)
+        {
+            return false;
+        }
+        if (getClass() != obj.getClass())
+        {
+            return false;
+        }
+        final ActionArgument other = (ActionArgument) obj;
+        if (actionArgumentList == null)
+        {
+            if (other.actionArgumentList != null)
+            {
+                return false;
+            }
+        }
+        else if (!actionArgumentList.equals(other.actionArgumentList))
+        {
+            return false;
+        }
+        if (name == null)
+        {
+            if (other.name != null)
+            {
+                return false;
+            }
+        }
+        else if (!name.equals(other.name))
+        {
+            return false;
+        }
+        if (required != other.required)
+        {
+            return false;
+        }
+        if (type != other.type)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1126)